ftd.html

```diff
@@ -15,6 +15,7 @@
     </script>
     <style>
         __ftd_css__
+        __ftd_element_css__
```

Comment:
Don't create this, and append `__ftd_css__` and `__ftd_element_css__` in Rust side.